### PR TITLE
Progressbar accelerometer

### PR DESCRIPTION
**Pull request by jposada202020**
This adds an example on using the progressbar to show actual accelerometer Data

## Expected results

Using Data taken from a MSA301 and included in the file to be sensor independent

### CODE REVIEW

File: examples/progressbar_accelerometer.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    for val in fake__accel_data:
+        if val[0] > 0:
+            left_horizontal_bar.value = 0
+            right_horizontal_bar.value = val[0]
+        if val[0] < 0:
+            left_horizontal_bar.value = -val[0]
+            right_horizontal_bar.value = 0
+        if val[0] == 0:
+            right_horizontal_bar.value = 0
+            left_horizontal_bar.value = 0
```
Comment (dhalbert) [inline examples/progressbar_accelerometer.py]: This case could be merged into one of the cases above by using, say, `val[0] >= 0`.

File: examples/progressbar_accelerometer.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+        if val[1] > 0:
+            top_vertical_bar.value = val[1]
+            bottom_vertical_bar.value = 0
+        if val[1] < 0:
+            bottom_vertical_bar.value = -val[1]
+            top_vertical_bar.value = 0
+        if val[1] == 0:
+            top_vertical_bar.value = 0
+            bottom_vertical_bar.value = 0
```
Comment (dhalbert) [inline examples/progressbar_accelerometer.py]: Same idea: could be subsumed into one of the cases above.

**Review by dhalbert — CHANGES_REQUESTED** [review]
Nice example!

**Comment by jposada202020** [comment]
will do as suggested, thanks @dhalbert :)

File: examples/progressbar_accelerometer.py (hunk 2 above)
Comment (jposada202020) [reply examples/progressbar_accelerometer.py]: Solved in https://github.com/adafruit/Adafruit_CircuitPython_ProgressBar/pull/31/commits/bd229635af7396f07b110be45d84a18745addab3

File: examples/progressbar_accelerometer.py (hunk 1 above)
Comment (jposada202020) [reply examples/progressbar_accelerometer.py]: solved in https://github.com/adafruit/Adafruit_CircuitPython_ProgressBar/pull/31/commits/bd229635af7396f07b110be45d84a18745addab3

**Comment by jposada202020** [comment]
@dhalbert I did the changes as suggested. Thanks

**Review by dhalbert — APPROVED** [review]
Thanks!

**Pull request merged**